### PR TITLE
networkd: Add example to create MACVTAP and attach it to a link

### PR DESCRIPTION
**Pull request by ssahani**
Fixes: #4914

### CODE REVIEW

File: man/systemd.network.xml [hunk 1]
```diff
@@ -1490,6 +1490,17 @@ VRF=vrf1
 </programlisting>
     </example>
 
+    <example>
+      <title>MacVTap</title>
+      <para>MacVTap</para>
```
Comment (poettering) [inline man/systemd.network.xml]: can you extend the paragraph here a bit? Doesn't have to be too long, but should be at least one proper sentence with a verb

**Comment by ssahani** [comment]
Updated .

File: man/systemd.network.xml [hunk 2]
```diff
@@ -1490,6 +1490,18 @@ VRF=vrf1
 </programlisting>
     </example>
 
+    <example>
+      <title>MacVTap</title>
+      <para>This brings interface <literal>macvtap-test</literal> up and attach it to the
```
Comment (poettering) [inline man/systemd.network.xml]: the english isn't really in order. It should say "This brings up a network interface … and attaches it …"

Comment (ssahani) [reply man/systemd.network.xml]: OK

**Pull request merged**